server/proxy/grpcproxy/auth.go

```diff
@@ -18,98 +18,81 @@ import (
 	"context"
 
 	pb "go.etcd.io/etcd/api/v3/etcdserverpb"
-	"go.etcd.io/etcd/client/v3"
+	clientv3 "go.etcd.io/etcd/client/v3"
 )
 
 type AuthProxy struct {
-	client *clientv3.Client
+	auth pb.AuthClient
```

Comment:
suggest to rename to `authClient`

Reply:
Done. Thanks for your suggestions. I have tow questions in the discussions. If you are free, I hope you can help me.